lib/std/crypto/25519/field.zig

```diff
@@ -66,6 +89,29 @@ pub const Fe = struct {
         return s;
     }
 
+    /// Map a 64-bit big endian string into a field element
+    pub fn fromBytes64(s: [64]u8) Fe {
```

Comment:
> pub fn toBytes64(fe: Fe) [64]u8 {

is not useful?

Reply:
The field size is ~255 bits, so a 512 bits inverse map wouldn't be very useful :)

The purpose of a 2n->n bit map is to make biases negligible after reduction. Using a 512 bit input for 25519 is just theater since the field size is so close to a power of 2. But the H2C standard was designed to work with any curve, and defines a unique function for that map.